project-base/tests/FrontendApiBundle/Functional/Product/ProductTest.php

```diff
@@ -0,0 +1,131 @@
+use Shopsys\FrameworkBundle\Model\Product\ProductFacade;
+use Tests\FrontendApiBundle\Test\GraphQlTestCase;
+
+class ProductTest extends GraphQlTestCase
+{
+    /**
+     * @var \Shopsys\ShopBundle\Model\Product\Product
+     */
+    private $product;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    private $domain;
+
+    protected function setUp(): void
+    {
+        $this->domain = $this->getContainer()->get(Domain::class);
+        $productFacade = $this->getContainer()->get(ProductFacade::class);
+        $this->product = $productFacade->getById(1);
+
+        parent::setUp();
+    }
+
+    public function testProductDetailNameByUuid(): void
+    {
+        $query = '
+            query {
+                product(uuid: "' . $this->product->getUuid() . '") {
+                    name
+                }
+            }
+        ';
+
+        $arrayExpected = [
+            'data' => [
+                'product' => [
+                    'name' => t('22" Sencor SLE 22F46DM4 HELLO KITTY', [], 'dataFixtures', $this->domain->getDomainConfigById(Domain::FIRST_DOMAIN_ID)->getLocale()),
+                ],
+            ],
+        ];
+
+        $this->assertQueryWithExpectedArray($query, $arrayExpected);
+    }
+
+    public function testProductDetailWithAllAttributesByUuid(): void
+    {
+        $query = '
+            query {
+                product(uuid: "' . $this->product->getUuid() . '") {
+                    name,
+                    shortDescription,
+                    link,
+                    unit,
+                    availability,
+                    stockQuantity,
+                    categories {
+                      name
+                    },
+                    flags {
+                      name, rgbColor
+                    },
+                    price {
+                      priceWithVat,
+                      priceWithoutVat,
+                      vatAmount
+                    }
+                }
+            }
+        ';
+
+        $this->assertQueryWithExpectedArray($query, $this->getExpectedProductDetailWithAllAttributes());
+    }
+
+    /**
+     * @return array
+     */
+    private function getExpectedProductDetailWithAllAttributes(): array
+    {
+        $firstDomainLocale = $this->domain->getDomainConfigById(Domain::FIRST_DOMAIN_ID)->getLocale();
+        $shortDescription = t(
+            'Television LED, 55 cm diagonal, 1920x1080 Full HD, DVB-T MPEG4 tuner with USB recording and playback',
+            [],
+            Domain::FIRST_DOMAIN_ID,
+            $firstDomainLocale
+        );
+
+        return [
+            'data' => [
+                'product' => [
+                    'name' => t('22" Sencor SLE 22F46DM4 HELLO KITTY', [], 'dataFixtures', $firstDomainLocale),
+                    'shortDescription' => $shortDescription,
+                    'link' => 'http://webserver:8080/22-sencor-sle-22f46dm4-hello-kitty/',
```

Comment:
This should be resolved same way as in LocalizationHelper::getLocalizedPathOnFirstDomainByRouteName

Reply:
Fixed https://github.com/shopsys/shopsys/pull/1471/commits/f8c97aed253d218f8aefc7bcad5a64855ed5b782